Tools/scripts/freeze_modules.py

```diff
@@ -523,7 +523,7 @@ def regen_pcbuild(modules):
             continue
         pyfile = os.path.relpath(src.pyfile, ROOT_DIR).replace('/', '\\')
         header = os.path.relpath(src.frozenfile, ROOT_DIR).replace('/', '\\')
-        intfile = header.split('\\')[-1].strip('.h') + '.g.h'
+        intfile = os.path.splitext(os.path.basename(header))[0] + '.g.h'
```

Comment:
FYI @ericsnowcurrently I fixed a bug in your script here (should've been `removesuffix('.h')` rather than strip, but better to stick with `os.path` when dealing with paths anyway).

Reply:
Thanks for fixing this.  I was going to suggest using `ntpath` explicitly here, but in this case it really should be `os.path`. 🙂